scripts/lib/bump_version.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+from typing import Union
+
+import semver
+import yaml
+
+
+def _chart_yaml_path(chart_path: Union[Path, str]) -> Path:
+    """
+    Resolve the path to the Chart.yaml file,
+    whether it is given or just the directory containing it.
+    """
+    chart_path = Path(chart_path)
+    if chart_path.is_dir():
+        return chart_path / "Chart.yaml"
```

Comment:
Before return here I would check if the filepath exists and is readable. If not throw an error. Same below where the path is filepath.

Reply:
Ack, so one nit:
os.path.join(chart_path, "Chart.yaml")

Reply:
That's the same as what's there if using `pathlib`, I think.